xprof/xprof.rb.in

```diff
@@ -312,6 +312,17 @@ def env_tracers
     h['OMP_TOOL_LIBRARIES'] = File.join(PKGLIBDIR, 'libTracerOMPT.so')
   end
 
+  # Sample
+  if OPTIONS[:sample]
+    h['LTTNG_UST_SAMPLING'] = 1
+    h['LTTNG_UST_SAMPLING_ENERGY'] = 1
+
+    if OPTIONS[:'backend-names'].include?('ze')
+      h['ZES_ENABLE_SYSMAN'] = 1
+      h['ZET_ENABLE_METRICS'] = 1
```

Comment:
We used 'ZET_ENABLE_METRICS' when we were trying to use tools api, we can omit for now.

Reply:
Thanks will remove!